docs/src/cluster/bench-tps.md

```diff
@@ -10,7 +10,20 @@ First, install Rust's package manager Cargo.
 
 ```bash
 $ curl https://sh.rustup.rs -sSf | sh
-$ source $HOME/.cargo/env
+  $ source $HOME/.cargo/env
```

Comment:
Is it productive to have it in two places? Can we just point to the main repo README.md with a message like setup cargo, rust and system packages as described in <xx>.